src/main/resources/fabric.mod.json

```diff
@@ -27,10 +27,10 @@
   ],
 
   "depends": {
-    "fabricloader": "*",
```

Comment:
Fabric loader is still going to be required. I don't see a point in removing it.

Reply:
This file won't be loaded without Fabric Loader. I don't see a point in having it.